test/test_google_geocoder.rb

```diff
@@ -269,22 +269,35 @@ def test_language_response
 
   def test_too_many_queries
     response = MockSuccess.new
-    response.expects(:body).returns '{"status": "OVER_QUERY_LIMIT"}'
+    response.expects(:body).returns '{"status": "OVER_QUERY_LIMIT", "error_message": "quota exceeded!"}'
     url = "https://maps.google.com/maps/api/geocode/json?sensor=false&address=#{Geokit::Inflector.url_escape(@address)}"
     Geokit::Geocoders::GoogleGeocoder.expects(:call_geocoder_service).with(url).returns(response)
-    assert_raise Geokit::Geocoders::TooManyQueriesError do
+    err = assert_raise Geokit::Geocoders::TooManyQueriesError do
       Geokit::Geocoders::GoogleGeocoder.geocode(@address)
     end
+    assert_equal "quota exceeded!", err.message
+  end
+
+  def test_access_denied
+    response = MockSuccess.new
+    response.expects(:body).returns '{"status": "REQUEST_DENIED", "error_message": "access denied!"}'
+    url = "https://maps.google.com/maps/api/geocode/json?sensor=false&address=#{Geokit::Inflector.url_escape(@address)}"
```

Comment:
LMK if you want me to tweak the line lengths in these tests. I'm assuming they're fine as-is, since there are other similarly-sized lines here already.